### ISSUE TITLE
Error on yarn (missing ./assets under src/js?)

### ISSUE THREAD

**Issue by lfscheidegger** [josdejong/jsoneditor#785]
Hi!

I'm having an issue building an app on yarn that uses jsoneditor `7.0.2`. When importing jsoneditor like so:

```
import JSONEditor from 'jsoneditor';
```

Yarn complains about missing `jsonlint` and `selectr` under `./assets` in `src/js`. In more detail:

```
    ERROR in ./node_modules/jsoneditor/src/js/util.js
    Module not found: Error: Can't resolve './assets/jsonlint/jsonlint' in '/Users/luiz/src/nextdoor.com/apps/nextdoor/frontend/node_modules/jsoneditor/src/js'
     @ ./node_modules/jsoneditor/src/js/util.js 5:0-49 303:14-22 304:4-12
     @ ./node_modules/jsoneditor/src/js/JSONEditor.js
     @ ./node_modules/jsoneditor/index.js
     @ [redacted]
    
    ERROR in ./node_modules/jsoneditor/src/js/showTransformModal.js
    Module not found: Error: Can't resolve './assets/selectr/selectr' in '/Users/luiz/src/nextdoor.com/apps/nextdoor/frontend/node_modules/jsoneditor/src/js'
     @ ./node_modules/jsoneditor/src/js/showTransformModal.js 3:0-46 158:37-44 159:40-47 160:35-42 161:35-42 162:38-45
     @ ./node_modules/jsoneditor/src/js/textmode.js
     @ ./node_modules/jsoneditor/src/js/JSONEditor.js
     @ ./node_modules/jsoneditor/index.js
     @ [redacted]
```

I went looking in `node_modules/jsoneditor/src/js` and indeed, I can't find an `assets` directory. I'm going to try this on an older version of `jsoneditor` and can report back.

Thanks!

**Comment by josdejong** [comment]
Thanks for reporting. It could be that this issue has the same cause as #783, which is probably fixed in `v7.0.3`. Can you give that a try Luiz?

**Comment by lfscheidegger** [comment]
@josdejong thanks! `7.0.3` fixed it. Just a note, though, as you mentioned in #783 that this had been broken in `7.0.0`. I tried installing `6.4.1`, but still had the same problem. I also verified the contents of `index.js` in `6.4.1`, and indeed it points to `src`, not `dist`:

```
[luiz@:~/temp]$ cat node_modules/jsoneditor/index.js 
module.exports = require('./src/js/JSONEditor');
```

Anyway, probably doesn't matter since it's fixed in `7.0.3`, just wanted to give you the FYI :)
Thanks again!
